test/migrate.test.js

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env node
+
+import { describe, expect, test, beforeEach } from 'vitest';
+import { commandSync } from 'execa';
+import { Project } from 'fixturify-project';
+import { readFileSync } from 'node:fs';
+
```

Comment:
instead of calling rehearsal cli with `yarn` we need to be sure we are pulling in the correct binary eg. just copy `cli.test.js` https://github.com/rehearsal-js/smoke-test/blob/master/test/cli.test.js#L14-L23

Reply:
Refactored this to use a similar `run` command. 

Usages of `run` util  between `cli.test.js` and `migrate.test.js` and `grpah.test.js`:
- The  cwd where the command is run.
- When we run against a project-fixture we must specify it's baseDir, because of the runtime peerDependencies on typescript, eslint, and prettier.